src/test/resources/postgres-conf-kek.json

```diff
@@ -3,5 +3,6 @@
   "port" : 7777,
   "database" : "postgres",
   "username" : "oaitest_mod_oai_pmh",
-  "password" : "oaiTest"
+  "password" : "oaiTest",
```

Comment:
is it important to rename it to `src/test/resources/postgres-conf-kek.json`

Reply:
This file was removed.